packages/modal/stories/Modal.stories.tsx

```diff
@@ -11,7 +11,6 @@ import {
   LargeDialogModalWithFooter,
   FullscreenModal
 } from "../../index";
-import ModalStoryContainer from "./helpers/ModalStoryContainer";
```

Comment:
Make sure to remove this ModalStoryContainer file now that we don't need it anymore :)

Reply:
Hi @nataliepina , I have made the required changes